### ISSUE TITLE
Other language

### ISSUE THREAD

**Issue by adriannedbailo** [rsimmons/voracious#57]
Can I use it for learning other language?

**Comment by gbdsantos** [comment]
Yes, you can use, although the application focuses on the Japanese language

**Comment by artemmezh** [comment]
https://github.com/artemmezh/voracious/projects/1#card-82252403
